### DISCUSSION TITLE
Custom Field without database column

### DISCUSSION THREAD

**Discussion by fatihdurmaz** [filamentphp/filament#12371] [Help] [answered]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

I want to make changes within a form using a field that is not in the database. Based on the toggle value, I will hide or show another field, but the toggle itself will not be in the database.

                Toggle::make('kismi_teklif') // not in database just form 
                    ->live()
                    ->onColor('success')
                    ->offColor('danger')
                    ->label('İhale Kısmi Teklife Açık Mı?'),

                TextInput::make('kisim_sayisi')
                    ->label('Kısım Sayısı')
                    ->numeric()
                    ->visible(fn (Get $get) => $get('kismi_teklif') === true),

**Comment by leandrocfe** [comment, marked as answer]
https://filamentphp.com/docs/3.x/forms/advanced#preventing-a-field-from-being-dehydrated
?

**Reply by fatihdurmaz** [reply]
Thank you very much
